src/util/fallbackRenderer.tsx

```diff
@@ -0,0 +1,17 @@
+import { FallbackProps } from 'react-error-boundary';
+import { Alert, Button, Code, Text } from '@mantine/core';
+
+// Call resetErrorBoundary() to reset the error boundary and retry the render.
+export function fallbackRender({ error, resetErrorBoundary }: FallbackProps) {
+  return (
+    <Alert variant={'light'} color={'red'} title={'🚨 Houston, we have a problem...🚨'}>
+      <Text>An error with the following message was thrown:</Text>
+      <Code block my={'md'}>
+        {error.message}
+      </Code>
+      <Button onClick={resetErrorBoundary} mt={'md'}>
+        Try reloading property tree
```

Comment:
OBS! This button will not always reset the property tree, it depends on the provided reset function.  So this text should be changed, either to something more general or something that could be configured from the properties. 

Since we don't expect users to see this, maybe just "Reset" is fine?
```suggestion
        Reset
```

Reply:
Perhaps Reload ?